### DISCUSSION TITLE
Feedback for course "SSH and SCP"

### DISCUSSION THREAD

**Discussion by ludelafo** [heig-vd-dai-course/heig-vd-dai-course#115] [Polls]
Can you let us know what was easy and what was difficult for you during this chapter?

This will help us to improve the course and adapt the content to your needs. If we notice some difficulties, we will come back to you to help you.

You can use reactions to express your opinion on a comment! 🚀

Vous pouvez aussi nous laisser votre avis en français, il n'y a pas de souci !

**Comment by lucaslattion** [comment]
The command ssh-copy-id does not exist on Windows.
I used the following command instead:
type .\demo_ed25519.pub | ssh -p 2222 daistudent@localhost "cat >> .ssh/authorized_keys"

**Reply by ludelafo** [reply]
Thank you for the heads up!

Just to be sure, which environment are you using? Powershell, Command Prompt, WSL (if so, which distribution?), GitBash, ...?

Were the other commands available in your environment?

I will try to update the course accordingly.

**Comment by ludelafo** [comment]
All previous comments have been marked as hidden (outdated) for the start of the new DAI semester.
